libs/core/src/components/pds-switch/docs/pds-switch.mdx

```diff
@@ -102,7 +76,7 @@ Displays additional descriptive text underneath the main label.
       label="Remember me"
       name="pds-switch-helper"
       type="checkbox"
```

Comment:
Ned to remove `type` from the examples, since the property has been removed

Reply:
@ju-Skinner i think it's still needed on the underlining input though, no? how else would it be a checkbox?